README.rst

```diff
@@ -47,15 +47,15 @@ The following steps presuppose that you have Anaconda or Miniconda installed on
   This is needed to activate the conda environment.
 | For further information please visit the following website:
 | https://github.com/conda/conda/blob/master/CHANGELOG.md#440-2017-12-20
-| Set TOPAZ_ACTIVATION_CMD variable in the Packages section. 
-| For example: ``TOPAZ_ACTIVATION_CMD = conda activate topaz``
-| This will activate the conda environment with the default name topaz.
+| Set TOPAZ_ENV_ACTIVATION variable in the Packages section.
```

Comment:
I think this is optional....should we mention that?

Reply:
True, I just didn't want to change much prévious code. Ps. You are looking at the closed pr.